charts/auth-km-jobs/templates/cronjob.yml

```diff
@@ -21,6 +21,9 @@ spec:
             {{- end }}
         spec:
           serviceAccountName: {{ .Values.serviceAccount.name }}
+          {{- if .Values.imagePullSecretNames }}
```

Comment:
This is a `job` did you verify that this is a correct object spec (it should be OK)?

Reply:
We spoke about this previously and you said I should add it. It renders properly but I didn't try to install it

Reply:
Ok, great, thanks. I just wanted to be sure that is doesn't break anything.